third_party/blink/renderer/core/brave_page_graph/requests/tracked_request.cc

```diff
@@ -16,25 +21,30 @@ namespace brave_page_graph {
 TrackedRequest::~TrackedRequest() = default;
 
 // Constructor for when we see the outgoing request first.
-TrackedRequest::TrackedRequest(const InspectorId request_id,
+TrackedRequest::TrackedRequest(PageGraphContext* page_graph_context,
+                               const InspectorId request_id,
                                GraphNode* requester,
                                NodeResource* resource,
                                const String& resource_type)
-    : request_id_(request_id),
-      resource_type_(resource_type),
-      resource_(resource) {
-  CHECK(requester != nullptr);
-  CHECK(resource != nullptr);
+    : page_graph_context_(page_graph_context),
+      request_id_(request_id),
+      resource_type_(resource_type) {
+  DCHECK(page_graph_context_);
+  DCHECK(requester);
+  DCHECK(resource);
```

Comment:
why CHECK is now DCHECK

Reply:
these are mostly sanity checks, not runtime checks.